### ISSUE TITLE
[MultiSelect] Error: Cannot have duplicate keys in a keyed each

### ISSUE THREAD

**Issue by skygate2012** [IBM/carbon-components-svelte#358]
It seems MultiSelect has trouble with certain words, and I can't seem to find the pattern. Can you reproduce this issue with the following snippet?

`<MultiSelect items={[{id: 0,text: 'Harassment'},{id: 1,text: 'Workplace Violence'},{id: 2,text: 'Conflicts of Interest'}]} />`

**Comment by metonym** [comment]
Wow – this is a nasty bug.

```svelte
{#each filterable ? filteredItems : sortedItems as item, i (item.id || i)}
```

I believe the duplicate key error is caused by the id `0` being falsy.

A quick workaround would be to use strings for the item ids.

**Comment by skygate2012** [comment]
@metonym Mindblowing.. totally haven't thought about that!

**Comment by metonym** [comment]
This is fixed in [v0.20](https://github.com/IBM/carbon-components-svelte/releases/tag/v0.20.0).

Item ids in ComboBox, Dropdown, MultiSelect components can be numbers.
